src/igvfd/schemas/biosample.json

```diff
@@ -254,18 +254,18 @@
                 "decline"
             ]
         },
-        "biosample_ontology": {
-            "title": "Biosample ontology term",
-            "description": "Ontology term of the biosample.",
+        "biosample_term": {
+            "title": "Biosample term",
+            "description": "Term of the biosample.",
```

Comment:
i think the description should mention assay. so it would be something like "schema for an ontological term identifying a biosample."

Reply:
done